cluster/search/client.py

```diff
@@ -81,7 +81,7 @@ def count(self, query, **options):
         })
         return self.es.count(index=self.index, body=options)["count"]
 
-    def search(self, query, start_date=None, end_date=None, limit=100, only_documents=True, **kwargs):
+    def search(self, query, source=None, start_date=None, end_date=None, limit=100, only_documents=True, **kwargs):
```

Comment:
Wildcard doesn't seem to work for me, did the part where it was getting added look correct? Not sure what's up with that